lectures/02.intro_ue4.md

```diff
@@ -566,6 +570,8 @@ grep -r "public IModuleInterface" /Engine/Source/ | wc -l
 * There are few really important classes
     - `UObject` is the father of all; provides reflection and garbage collection
     - `AActor` is the parent of everything placeable in the world. Derived from `UObject`
+	- `APawn` is the parent of everything possessable by players or AI. Derived from `AActor`
+	- `AController` can possess a Pawn to control its actions. Derived from `UObject`

```

Comment:
I believe we deliberately left these out in order to not confuse people. They are better explained in architecture UE4

Reply:
I can't find anything about APawn and AController. Also we can explain it in more details.